doc/source/programs/ogr_layer_algebra.rst

```diff
@@ -0,0 +1,169 @@
+.. _ogr_layer_algebra:
+
+================================================================================
+ogr_layer_algebra.py
+================================================================================
+
```

Comment:
```suggestion

.. versionadded:: 3.6

```

Reply:
Added this in https://github.com/kssvrk/gdal/commit/0a8393cee671b57f498c051b3ba83a1c5401f6c4